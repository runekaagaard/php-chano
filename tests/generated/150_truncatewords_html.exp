<p>one <a href="#">two - three <br>four ...</a></p>