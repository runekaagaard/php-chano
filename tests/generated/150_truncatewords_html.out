Fatal error: Call to undefined method DtlIter::truncatewords_html() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/150_truncatewords_html.php on line 4